runtimerevolution/web3-roulette
# [WEB3-27] User empty state page

# Description

This PR adds the content for when there are no giveaways to present to a non-admin user.

Ticket: https://runtime-revolution.atlassian.net/browse/WEB3-27

## Type of change

- New feature (non-breaking change which adds functionality)

## How Has This Been Tested?

1. Remove any giveaway present in the database
2. Login as a normal user - update user role in the login card component
3. Verify the content of the landing page
4. Re-login as an admin, verify the difference

## Checklist:
